dev/com.ibm.ws.org.codehaus.jackson/bnd.bnd

```diff
@@ -12,7 +12,7 @@
 bVersion=1.0
 
 Bundle-Name: Jackson JSON processor 1.6.2
-Bundle-SymbolicName:  com.ibm.ws.org.codehaus.jackson.1.6.2
```

Comment:
I don't think you need to change the bundle symbolic name, just the libs that the bundle actually points to.

If you don't change the bundle symbolic name, then you also wouldn't need to modify ```dev/com.ibm.ws.jaxrs.2.0.common/bnd.bnd```